src/LibVLCSharp/Shared/RendererDiscoverer.cs

```diff
@@ -191,9 +188,6 @@ public class RendererItem : Internal
         /// <param name="disposing">true if called from a method</param>
         protected override void Dispose(bool disposing)
         {
-            if (IsDisposed || NativeReference == IntPtr.Zero)
```

Comment:
If you're just calling `base.Dispose(disposing)`,  you could jut remove the override altogether

Reply:
Done